packages/dev/core/src/Lights/simpleVolumetricSpotLight.ts

```diff
@@ -0,0 +1,106 @@
+import { SpotLight } from "./spotLight";
+import { Scene } from "core/scene";
+import { Axis } from "core/Maths/math.axis";
+import { CreateCylinder } from "core/Meshes/Builders/cylinderBuilder";
+import { Mesh } from "core/Meshes/mesh";
+import { ShaderMaterial } from "core/Materials/shaderMaterial";
+import "../Shaders/simpleVolumetricSpot.fragment";
+import "../Shaders/simpleVolumetricSpot.vertex";
+import { Observer, RenderTargetTexture, TmpVectors } from "..";

```

Comment:
it is not allowed to import from a folder, import every element from their respective files.